Formula/kubecfg.rb

```diff
@@ -22,7 +22,8 @@ def install
     cd "src/github.com/kubecfg/kubecfg" do
       system "make", "VERSION=v#{version}"
       bin.install "kubecfg"
-      pkgshare.install Dir["examples/*"], "testdata/kubecfg_test.jsonnet"
+      pkgshare.install (Pathname.pwd/"examples").children
+      pkgshare.install (Pathname.pwd/"testdata").children
```

Comment:
Sorry, that was more complicated than necessary. I actually edited my suggestion shortly after writing it.
```suggestion
      pkgshare.install Pathname("examples").children
      pkgshare.install Pathname("testdata").children
```